steamtinkerlaunch

```diff
@@ -26190,6 +26204,22 @@ function restoreGtkCss {
 	fi
 }
 
+function checkSteamDeckLastVers {
+	# This function just makes sure that the 'lastvers' file exists at the defined path
+	# If it does not we set it and clear the deps to ensure it starts downloading the libs at the right version
+	if ! [ -f "$STLSTEAMDECKLASTVERS" ]; then
+		clearDeckDeps
```

Comment:
Could this potentially cause problems? Just thinking out loud.

If we clear the dependencies and then set `STLSTEAMDECKLASTVERS` to the current version, wouldn't this cause the `checkSteamDeckSTLUpdated` check in `checkSteamDeckDependencies` to fail, and so we'd end up with no dependencies?

I _think_ the `-f "$(command -v "$DEPCMD")" && "$CHECKCMD" = "OK"` would catch this, and since the files don't exist we would fall through to downloading the dependencies. But it's just something I wanted to call out, it might be good to validate this in testing!

Reply:
Normally it should be covered by the `-f "$(command -v "$DEPCMD")" && "$CHECKCMD" = "OK"` check since it will fail because no libs will be available. It's currently how it already behaves.

I also thought about internet connection issues. However, since the clearDeps should **ONLY** happen after a fresh installation or an STL update internet should already be there to download the STL update in the first place.